README.md

```diff
@@ -23,33 +23,45 @@ If you'd like to contribute to `modlunky2` here are some steps to setup your env
 ### Creating VirtualEnv
 In the root directory you can make a virtualenv. It will be excluded from commits by default
 ```console
-> python -m venv venv
+python -m venv venv
 ```
 
 ### Activate VirtualEnv
 
 You'll want to activate the virtual environment whenever you're testing any commands from this package
 
 #### Powershell
+
+please check here first https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_execution_policies?view=powershell-7
```

Comment:
Maybe we should just remove the whole section on how to use a virtualenv and just link to the comprehensive docs: https://docs.python.org/3/library/venv.html#creating-virtual-environments